<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FactuurVanTicket</class>
 <widget class="QDialog" name="FactuurVanTicket">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1001</width>
    <height>615</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableView" name="tvTeFactureren">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>656</width>
     <height>431</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QListWidget" name="lstwTicket">
   <property name="geometry">
    <rect>
     <x>675</x>
     <y>30</y>
     <width>316</width>
     <height>546</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOvernemen">
   <property name="geometry">
    <rect>
     <x>287</x>
     <y>560</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Overnemen</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAnuleren">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>560</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Anuleren</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbTicketNr">
   <property name="geometry">
    <rect>
     <x>675</x>
     <y>10</y>
     <width>316</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>lbTicketNr</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbToaal">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>580</y>
     <width>141</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>lbToaal</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbTotaal">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>580</y>
     <width>141</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Totaal:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
